lib/modules/manager/gradle-wrapper/utils.ts

```diff
@@ -21,16 +21,17 @@ export function gradleWrapperFileName(): string {
   return './gradlew';
 }
 
-export async function prepareGradleCommand(): Promise<string | null> {
-  const gradlewFile = gradleWrapperFileName();
+export async function prepareGradleCommand(
```

Comment:
looks like a wrong merge, we don't need he argument, but maybe we need the relative path

Reply:
For `statLocalFile()` to work on a subfolder, the path to `gradlew` in that subfolder is needed.